eppo_client/constants.py

```diff
@@ -1,5 +1,3 @@
 # poller
-SECOND_MILLIS = 1000
-MINUTE_MILLIS = 60 * SECOND_MILLIS
-POLL_JITTER_MILLIS = 30 * SECOND_MILLIS
-POLL_INTERVAL_MILLIS = 5 * MINUTE_MILLIS
+POLL_JITTER_SECONDS = 2
+POLL_INTERVAL_SECONDS = 10
```

Comment:
Any reason not to do 30 like our other SDKs?

Reply:
golang is ten seconds - is 30 standard?